paper.md

```diff
@@ -167,15 +167,15 @@ Data consumption is not as streamlined, as disconnected data portals require a l
 Reuse of data and other sizable or binary files is not as transparent, as data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development;
 And scientific processes are not as reproducible, because data provenance, the information of how a digital file came to be, is often incomplete and rarely automatically captured.
 DataLad aims to solve these issues by providing streamlined, transparent management of code, data, computing environments, and their relationship.
-It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed, unified access to all elements of scientific project, and it enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
+It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed, unified access to all elements of scientific project, and it enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that enables automatic re-execution.
```

Comment:
```suggestion
It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed, unified access to all elements of scientific project, and it enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
```
I would keep it at `afford`.  On the difference between the two https://wikidiff.com/afford/enable#:~:text=is%20that%20enable%20is%20to,be%20able%20or%20rich%20enough. says

As verbs the difference between enable and afford is that enable is to give strength or ability to; to make firm and strong while afford is to incur, stand, or bear without serious detriment, as an act which might under other circumstances be injurious;—with an auxiliary, as can, could, might, etc; to be able or rich enough.

And since not everything what was `datalad run` could be `rerun` or even re-executed with a dance around  (users could have specified outside resources etc), I think `afford` makes it closer to the state of things.

Reply:
since this just a reversal of the change, I will commit this one and will leave the other two "pending"